src/screens/User/OtherProfile.js

```diff
@@ -174,6 +178,42 @@ class OtherProfile extends Component {
 			</View>
 		);
 	}
+
+	async openApp(profile, name) {
+		if (!profile) {
+			Alert.alert(`${ name } has not added his LinkedIn profile yet.`);
+		}
+		else {
+			let url = `linkedin://in/${ profile }`;
+
+			console.log(`URL: ${ profile }`);
+			// then have to check if the link can be handle
+			const canOpenLink = await Linking.canOpenURL(url);
+
+			if (canOpenLink)
+				await Linking.openURL(url);
+			else
+				await Linking.openURL(`https://linkedin.com/in/${ profile }`);
+		}
+	}
+
+	async openEmail(to, name) {
+		if (!to) {
+			Alert.alert(`No email for ${ name } has been found.`);
```

Comment:
Consider using the `type: "error"` for alert. Also, remove spacing around curly braces for interpolation